### ISSUE TITLE
Dockerfile is correct?

### ISSUE THREAD

**Issue by bscheshirwork** [Codeception/Codeception#5728]
#### What are you trying to achieve?
I trying to compile my own version of Codeception. But I can't passin install composer step without copy `tests` folder (after 47 line):
https://github.com/Codeception/Codeception/blob/9c69ec708f15140be20341d342c330d68fb04197/Dockerfile#L47-L48
```
# Add test-code
COPY ./tests /repo/tests
```
in mainline all is work fine?

**Comment by Naktibalda** [comment]
What error do you get?

**Comment by bscheshirwork** [comment]
```
  [RuntimeException]                                                                                    
  Could not scan for classes inside "tests/cli/_steps" which does not appear to be a file nor a folder  
 ```

```sh
...

monolog/monolog suggests installing graylog2/gelf-php (Allow sending log messages to a GrayLog2 server)
monolog/monolog suggests installing sentry/sentry (Allow sending log messages to a Sentry server)
monolog/monolog suggests installing doctrine/couchdb (Allow sending log messages to a CouchDB server)
monolog/monolog suggests installing ruflin/elastica (Allow sending log messages to an Elastic Search server)
monolog/monolog suggests installing php-amqplib/php-amqplib (Allow sending log messages to an AMQP server using php-amqplib)
monolog/monolog suggests installing ext-amqp (Allow sending log messages to an AMQP server (1.0+ required))
monolog/monolog suggests installing ext-mongo (Allow sending log messages to a MongoDB server)
monolog/monolog suggests installing mongodb/mongodb (Allow sending log messages to a MongoDB server via PHP Driver)
monolog/monolog suggests installing aws/aws-sdk-php (Allow sending log messages to AWS services like DynamoDB)
monolog/monolog suggests installing rollbar/rollbar (Allow sending log messages to Rollbar)
monolog/monolog suggests installing php-console/php-console (Allow sending log messages to Google Chrome)
Writing lock file
Generating optimized autoload files

                                                                                                        
  [RuntimeException]                                                                                    
  Could not scan for classes inside "tests/cli/_steps" which does not appear to be a file nor a folder  
                                                                                                        

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...

The command '/bin/sh -c composer install --prefer-dist --optimize-autoloader' returned a non-zero code: 1

```

**Comment by Naktibalda** [comment]
@OneEyedSpaceFish is working on this issue.

**Comment by bscheshirwork** [comment]
@OneEyedSpaceFish 
```
COPY ./composer.json /repo/composer.json 
# Add test-code
COPY ./tests /repo/tests
RUN composer install --prefer-dist --no-interaction --optimize-autoloader --apcu-autoloader 
```
must work fine

**Comment by OneEyedSpaceFish** [comment]
PR opened.
In the meanwhile if you want to pull new images, feel free to use mine (rebuilt the missing versions with fixed Dockerfile)
https://hub.docker.com/r/oneeyedspacefish/codeception/tags

**Comment by Naktibalda** [comment]
Fixed by #5917
Please test codeception/codecetion:4.1.5
